Rakefile

```diff
@@ -8,3 +8,16 @@ task :download, :download_flag, :extension do |_t, args|
   extension = args[:extension].to_s
   Downloader.new.download_with_options(download_flag, extension)
 end
+
+desc 'Download files for docker'
+task :docker do |_t|
+  download_flag = if ENV['EXT'] != ''
+                    :ext
+                  else
+                    ENV['FlAG'].to_sym
+                  end
+  extension = ENV['EXT'].to_s
+  downloader = Downloader.new
+  downloader.s3_key_writer
```

Comment:
I woud extract this method to separate class and make this singleton

Something like

`Config.init_s3_from_env`